nucypher/policy/maps.py

```diff
@@ -130,6 +127,9 @@ def from_bytes(cls, data: bytes):
         destinations = {u: k for u, k in ursula_and_kfrags}
         return cls(threshold, hrac, destinations)
 
+    def __eq__(self, other):
```

Comment:
I don't see any problems with it. Where is it used, in tests?

Reply:
One thing though: if there's a different order in `destinations`, it will lead to the equality returning `False`, while the destinations might be the same.

Reply:
It is used for tests.

> if there's a different order in destinations, it will lead to the equality returning False

Good point - perhaps I'll modify it to compare `threshold`, `destinations`, and `hrac` individually instead.

Reply:
✔️